scripts/drag_drop_handler.lua

```diff
@@ -0,0 +1,37 @@
+-- This script gets two parameters in its global scope:
+-- filename:   The full path for the file that was dropped on the application.
+--             Example:  C:/OpenSpace/openspace.cfg
+-- basename:   Only the name of the actual file with extension, but without the full rest
+--             of the path.
+--             Example:  openspace.cfg
+-- extension:  The extention of the file
+--             Example: cfg
+--
+-- From this script, we need to return the script that we want to be executed in response
+-- to the drag event.  If we don't want anything to happen, don't return anything or
+-- return an empty string
+
+if filename == nil or filename == "" or
+   basename == nil or basename == "" or
+   extension == nil or extension == "" then
+  do return "" end
+end
+
+-- Lua doesn't enjoy \ that are used by Windows extensively. So we convert all \ into /
+filename = filename:gsub("\\", "/")
+basename = basename:gsub("\\", "/")
+basename_without_extension = basename:sub(0, #basename - extension:len())
+
+if     extension == ".jpg" or extension == ".jpeg" or extension == ".tif" or
+       extension == ".png" then
```

Comment:
I could see this section being added-to in the future with new file types. How about adding functions to check for file types, like:
`if isImage() then`
The function itself would test for the 4 file image types (a list which may expand) and make the evaluation of file types cleaner.

Reply:
Good idea!